index.rst

```diff
@@ -38,80 +31,63 @@ The Vera Rubin Observatory control system is highly distributed, containing mult
 In order to achieve reliable operation it is fundamental to guarantee that observers must be able to, at any point, identify what set of configurations a component is using, what sets are available to be loaded, and how to load the configuration they choose.
 During commissioning and engineering runs, users will also be interested in building, validating and verifying new sets of configurations.
 
-The process of building new or modifying configurations will vary considerably for each component.
+The content needing to be configured and how that content is derived varies considerably for each component.
```

Comment:
Sorry but I prefer the original version... I think the wording here makes it a bit confusing.

Reply:
but the first one is now incorrect. The process of modifying configurations does NOT vary considerably between components. What goes in the files certainly does. How about, "The contents contained in configuration files varies considerably between components but the process of updating the files is standardized."

Reply:
oh! I see what you mean... How about?

```
Although the configuration data structure (e.g. schema) vary considerably between components, the process of updating them is standardized.
```

Reply:
I'm not very happy with either. Are you are saying "the process of specifying configuration will vary considerably..."? 

Which I'm still uneasy about. I think the process of specifying which_named *set* of configuration parameters to use is standardized in the start command. It's the process of specifying the parameters in a named configuration that is somewhat variable.

Reply:
i'm trying (albeit clearly unsuccessfully) to say that we've basically standardized how we do configuration. There are lots of CSCs, they all have different things/values that get configured, but they all follow the same principles. Same commands, same repo structure, same file format, same workflow to make changes etc.